src/redux/sagas/utils/expenditureMulticallData.ts

```diff
@@ -136,3 +137,49 @@ export const getMulticallDataForPayout = (
 
   return encodedMulticallData;
 };
+
+const convertNumberToBytes32 = (number: number) =>
+  hexZeroPad(hexlify(number), 32);
+
+const mask = [false, true];
```

Comment:
I believe this mask is specific for a particular multicall, so it should be put inside the relevant method or renamed